src/actions/profile-view.js

```diff
@@ -76,9 +76,24 @@ export function showThread(threadIndex: ThreadIndex): Action {
   };
 }
 
-export function changeCallTreeSearchString(searchString: string): Action {
+export function changeCurrentCallTreeSearchString(
```

Comment:
The logic for the actions is:
* we have one to change the current search string, the one the user is currently typing
* then we have one to _commit_ it to the search stack. This is when the user presses `enter`.
* at last we have one to remove it